package.json

```diff
@@ -9,20 +9,23 @@
     ]
   },
   "dependencies": {
-    "blockly": "1.0.0",
-    "cache-element": "2.0.1",
-    "choo": "4.0.1",
+    "@robotopia/agent-runtime": "^1.0.0",
+    "@robotopia/assets-loader": "^1.0.0",
+    "@robotopia/choo-game": "^1.0.0",
+    "@robotopia/clock": "^1.0.0",
+    "blockly": "^1.0.0",
+    "cache-element": "^2.0.1",
+    "choo": "^4.0.1",
     "choo-component": "git+https://github.com/ohgoodlord/choo-component.git",
-    "classnames": "2.2.5",
-    "cuid": "1.3.8",
-    "esper.js": "0.1.7",
-    "immutability-helper": "2.0.0",
-    "lodash": "4.16.4",
+    "classnames": "^2.2.5",
+    "cuid": "^1.3.8",
+    "esper.js": "^0.1.7",
+    "immutability-helper": "^2.0.0",
+    "lodash": "^4.16.4",
```

Comment:
Change dependency requirements to be more permissive so if a submodule can reuse dependencies